src/Microsoft/Silk.NET.XInput/XInput.cs

```diff
@@ -12,7 +12,7 @@ public partial class XInput
     {
         public static XInput GetApi()
         {
-             throw new NotImplementedException();
+            return new XInput(CreateDefaultContext("XInput1_3"));
```

Comment:
We usually use a library name container instead of hardcoding the name for platform compat. Do we not want this for XInput for some reason?

Reply:
I did this because the same thing is done in `XAudio`, and also I couldn't find a library container for XInput, not one rider could find anyway.